dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
-log.directory.desc=Location of the directory for log files. The default value is ${server.output.dir}/logs.
+log.directory.desc=You can use this attribute to set a directory for all log files, excluding the console.log file, but including FFDC. By default, logDirectory is set to the LOG_DIR environment variable. The default LOG_DIR environment variable path is WLP_OUTPUT_DIR/serverName/logs. Avoid trouble: Use the LOG_DIR environment variable or com.ibm.ws.logging.log.directory property, except in cases where you are trying to change the configuration dynamically after startup.
 
 message.file.name=Message file name
-message.file.name.desc=Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log.
+message.file.name.desc=Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log. This file always exists, and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages in addition to System.out and System.err. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log. Avoid trouble: Use the com.ibm.ws.logging.message.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 
 trace.file.name=Trace file name
-trace.file.name.desc=Name of the file to which trace output will be written relative to the configured log directory. The default value is trace.log.
+trace.file.name.desc=Name of the file to which trace output will be written relative to the configured log directory. The default value is trace.log. The trace.log file is only created if a traceSpecification is set including log levels below INFO. stdout is recognized as a special value, and causes trace to be directed to the original standard out stream. Avoid trouble: Use the com.ibm.ws.logging.trace.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 #
 # Console log level
 #
 console.level=Console log level
-console.level.desc=The logging level used to filter messages written to system streams. The default value is audit.
+console.level.desc=The logging level used to filter messages written to system streams. The valid values are INFO, AUDIT, WARNING, ERROR, and OFF. By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set) or AUDIT. Note: Before you change this value, consider the information in section "Unable to interact with the Liberty server after modifying the console log level settings" in the topic Developer Tools known restrictions. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change
"By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set) or AUDIT. Note: Before you change this value, consider the information in section "Unable to interact with the Liberty server after modifying the console log level settings" in the topic Developer Tools known restrictions. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup."
to
"By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set), or AUDIT. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or the com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup."

Is it okay to remove the note here? We want to avoid referencing documentation in our messages, if possible.

Reply:
the note is still important... if we don't want to reference the note, we need to put something here to say roughly what it says.

Reply:
@donbourne @kbaillie Is this desc going to be hover text?

Reply:
@kbaillie Okay to leave the note in there, but we may need to review this message regularly to ensure that the file that it references is removed/changed at any point. 

So, change
"By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set) or AUDIT. Note: Before you change this value, consider the information in section "Unable to interact with the Liberty server after modifying the console log level settings" in the topic Developer Tools known restrictions. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup."
to
"By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set), or AUDIT. Note: Before you change this value, consider the information in the section "Unable to interact with the Liberty server after modifying the console log level settings" in the topic Developer Tools known restrictions. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or the com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup.